### PR TITLE
COL-44 - Set up bookmarklet structure

### PR DESCRIPTION
**Pull request by nicolaasmatthijs**
- Add better styling to the bookmarklet
- Add ability to close modal
- Prevent modal from being opened multiple time
- Show success notification and link back to Asset Library
- Improve clickability of items from the page
- Show empty messages when no items can be found on the page

https://jira.ets.berkeley.edu/jira/browse/COL-44

### CODE REVIEW

File: public/assets/css/bookmarklet.css [hunk 1]
```diff
@@ -57,20 +82,34 @@ h5:first-child {
...
+.collabosphere-item-container label {
+  cursor: pointer;
+}
+
+.collabosphere-items-empty {
+  margin-bottom: 25px;
+}
+
+.collabosphere-items-list:empty + .collabosphere-items-empty {
+  display: block !important;
```
Comment (christianv) [inline public/assets/css/bookmarklet.css]: @nicolaasmatthijs interesting, why did we need important here?

File: public/assets/css/main.css [hunk 2]
```diff
@@ -42,6 +38,15 @@ h3 {
   margin-top: 30px;
 }
 
+h1:first-child,
```
Comment (christianv) [inline public/assets/css/main.css]: @nicolaasmatthijs interesting that we're not using SASS

File: public/assets/css/bookmarklet.css (hunk 1 above)
Comment (nicolaasmatthijs) [inline public/assets/css/bookmarklet.css]: I thought I needed it to override the 'hide' class, but I found a better approach and got rid of the `!important`

**Comment by nicolaasmatthijs** [comment]
Followed up on review. Ready for further review and merging.

**Pull request merged**